e2e/cypress/integration/bot_accounts/crud_spec.js

```diff
@@ -0,0 +1,252 @@
+            cy.get('[data-testid=saveSetting]').click();
+            cy.get('#clientError').should('not.be.visible');
+        });
+    });
+
+    it('MM-T1848 Delete Token', () => {
+        // # Visit the integrations
+        cy.visit(`/${newTeam.name}/integrations/bots`);
+
+        // * Check that the previously created bot is listed
+        cy.findByText(`Test Bot (@${botName})`).then((el) => {
+            // # Make sure it's on the screen
+            cy.wrap(el[0].parentElement.parentElement).scrollIntoView();
+
+            // # Click the 'Create token' button
+            cy.wrap(el[0].parentElement.parentElement).findByText('Create New Token').should('be.visible').click();
+
+            // # Add description
+            cy.wrap(el[0].parentElement.parentElement).find('input').click().type('description!');
+
+            // # Save
+            cy.get('[data-testid=saveSetting]').click();
+
+            // # Click Close button
+            cy.wrap(el[0].parentElement.parentElement).findByText('Close').should('be.visible').click();
+
+            cy.wrap(el[0].parentElement.parentElement).scrollIntoView();
+
+            // * Check that token is visible
+            cy.wrap(el[0].parentElement.parentElement).findByText(/Token ID:/).should('be.visible');
+
+            // # Click Delete button
+            cy.wrap(el[0].parentElement.parentElement).findByText('Delete').should('be.visible').click();
+
+            // * Validate that delete button is visible and click it
+            cy.get('#confirmModalButton').should('be.visible').click();
+
+            // * Check that token is not visible
+            cy.wrap(el[0].parentElement.parentElement).findByText(/Token ID:/).should('not.be.visible');
+        });
+    });
+    it('MM-T1849 Create a Personal Accees Token when email cofig is invalid', () => {
+        // # Make sure that email is now functioning
+        const newSettings = {
+            EmailSettings: {
+                SMTPServer: '',
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        // # Create a test bot and validate that token is created
+        const botUsername = `bot-${getRandomId()}`;
+        createBotInteractive(botUsername);
+        cy.get('#doneButton').click();
+
+        // # Add a new tocket to the bot
+        // * Check that the previously created bot is listed
+        cy.findByText(`Test Bot (@${botUsername})`).then((el) => {
+            // # Make sure it's on the screen
+            cy.wrap(el[0].parentElement.parentElement).scrollIntoView();
+
+            // # Click the 'Create token' button
+            cy.wrap(el[0].parentElement.parentElement).findByText('Create New Token').should('be.visible').click();
+
+            // # Add description
+            cy.wrap(el[0].parentElement.parentElement).find('input').click().type('description!');
+
+            // # Save
+            cy.get('[data-testid=saveSetting]').click();
```

Comment:
This test seems not to be doing any kind of check. Am I missing something?

Reply:
you are right. missed a few more lines :) added.